src/components/views/settings/ChangePassword.js

```diff
@@ -64,10 +64,15 @@ module.exports = React.createClass({
         };
     },
 
-    changePassword: function(old_password, new_password) {
-        var cli = MatrixClientPeg.get();
+    changePassword: function(oldPassword, newPassword) {
+        const cli = MatrixClientPeg.get();
 
-        var QuestionDialog = sdk.getComponent("dialogs.QuestionDialog");
+        if (this.props.disableConfirmation) {
```

Comment:
Add to proptypes please. Also if we can phrase it to avoid the double-negative disableConfirmation=false that would be good (I suggest 'confirm', defaulting to true).